### DISCUSSION TITLE
Are there plans to support ARM64? Windows 11 on ARM64

### DISCUSSION THREAD

**Discussion by joesdu** [git-for-windows/git#4110] [Q&A] [answered]
Are there plans to support ARM64? Windows 11 on ARM64

**Comment by rimrul** [comment, marked as answer]
Yes, there are plans to provide native builds for Windows 11 on ARM64. For a detailed outline of the progress towards that please see issue #3107.

**Comment by joesdu** [comment]
> Yes, there are plans to provide native builds for Windows 11 on ARM64. For a detailed outline of the progress towards that please see issue #3107.

Thanks very much for your work in this area.
